processor/deltatocumulativeprocessor/processor.go

```diff
@@ -109,27 +128,35 @@ func (p *Processor) Capabilities() consumer.Capabilities {
 }
 
 func (p *Processor) ConsumeMetrics(ctx context.Context, md pmetric.Metrics) error {
+	defer fatal.Recover(p.ctx)
```

Comment:
After looking at the recovers, I'm feeling kind of bad for having them here. If we are confident that we can catch the panics with our unit tests, we can leave those recovers out. If we keep them, we'd need a new test for the ConsumeMetrics function, exercising the behavior upon a panic.

Reply:
all panics unit-tested, fatal handling removed